jsk_rviz_plugins/src/pie_chart_display.cpp

```diff
@@ -359,6 +363,11 @@ namespace jsk_rviz_plugins
     
   }
   
+  void PieChartDisplay::updateCaption()
+  {
+    caption_ = caption_property_->getStdString();
```

Comment:
> btw, caption_property_->getStdString(); is initialized at https://github.com/jsk-ros-pkg/jsk_visualization/pull/636/files#diff-21915262cfae72cf9ac2af03ff4ea8e7R91, so if you worried about null caption name, then we can set at there.
If you think we do not have to define caption_, but can use name_, I do not have confident on modifying name_ because did not know how is this variable used

Probably `name_` is the name of display and cannot be changed, so I think `caption_` variable is necessary.
<img width="121" alt="screen shot 2016-09-12 at 10 21 26 pm" src="https://cloud.githubusercontent.com/assets/4310419/18437307/483df3c0-7937-11e6-9ef7-f6e88c45de4c.png">